ipa_test.go

```diff
@@ -0,0 +1,22 @@
+package verkle
+
+import (
+	"encoding/binary"
+	"math/big"
+	"testing"
+)
+
+func TestFromBytes(t *testing.T) {
+	var fr Fr
+
+	var beFortyTwo [8]byte
+	binary.BigEndian.PutUint64(beFortyTwo[:], 42)
+
+	FromBytes(&fr, beFortyTwo[:])
```

Comment:
This `FromBytes` method isn't used in `go-verkle` anywhere. It's used in `go-ethereum` for tree-key calculations.
Adding a test to test that it does what we expect.